reference/system-databases/metrics-schema.md

```diff
@@ -0,0 +1,179 @@
+
+* `Select`: The `select` type statement is executed.
+* `internal`: The internal SQL statement of TiDB, which is used to update the statistical information and get the global variables.
+
+The execution plan of the above statement is as follows:
+
+{{< copyable "sql" >}}
+
+```sql
+desc select * from metrics_schema.tidb_query_duration where value is not null and time>='2020-03-25 23:40:00' and time <= '2020-03-25 23:42:00' and quantile=0.99;
+```
+
+```
++------------------+----------+------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| id               | estRows  | task | access object             | operator info                                                                                                                                                                                          |
++------------------+----------+------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Selection_5      | 8000.00  | root |                           | not(isnull(Column#5))                                                                                                                                                                                  |
+| └─MemTableScan_6 | 10000.00 | root | table:tidb_query_duration | PromQL:histogram_quantile(0.99, sum(rate(tidb_server_handle_query_duration_seconds_bucket{}[60s])) by (le,sql_type,instance)), start_time:2020-03-25 23:40:00, end_time:2020-03-25 23:42:00, step:1m0s |
++------------------+----------+------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+From the above result, you can see that `PromQL`, `start_time`, `end_time`, and the value of `step`. During actual execution, TiDB calls the `query_range` HTTP API interface of Prometheus to query the monitoring data.
+
+You might find that during the range of [`2020-03-25 23:40:00`, `2020-03-25 23:42:00`], each label only has three time values. In the execution plan, the value of `step` is 1 minute, which is determined by the following two variables:
+
+* `tidb_metric_query_step`: The resolution step of the query. To get the `query_range` data from Prometheus, you need to specify `start`, `end`, and `step`. `step` uses the value of this variable.
```

Comment:
- How about using [the similar description in Prometheus's doc](https://prometheus.io/docs/prometheus/latest/querying/api/)? @reafans Please also help confirm. Thanks!

- Confirm whether its's `start`/`end` or `start_time`/`end_time` here.

```suggestion
* `tidb_metric_query_step`: The query resolution step width. To get the `query_range` data from Prometheus, you need to specify `start`, `end`, and `step`. `step` uses the value of this variable.
```

Reply:
Start_time/end_time is more accurate, I'll fix it in Chinese version.